cypress.config.js

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require('cypress');
```

Comment:
Cypress docs suggest moving from the json file to `cypress.config.js` or `cypress.config.ts`. I needed to do this so we could pipe down the `NODE_ENV` environment variable to the Cypress runner.